components/binary_sensor/cap1188.rst

```diff
@@ -0,0 +1,81 @@
+CAP1188 Capacitive Touch Sensor
+===============================
+
+.. seo::
+    :description: Instructions for setting up CAP1188 Capacitive Touch Sensor
+    :image: cap1188.jpg
+    :keywords: CAP1188
+
+.. _cap1188-component:
+
+Component/Hub
+-------------
+
+The ``cap1188`` sensor platform allows you to use your CAP1188
+(`datasheet <https://cdn-shop.adafruit.com/datasheets/CAP1188.pdf>`__,
+`Adafruit`_) Capacitive Touch Sensor with ESPHome. The :ref:`I²C <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/cap1188-full.jpg
+    :align: center
+    :width: 75%
+
+    CAP1188 Capacitive Touch Sensor. Image by `Adafruit`_.
+
+.. _Adafruit: https://learn.adafruit.com/adafruit-cap1188-breakout/overview
+
+.. code-block:: yaml
+
+    # Example configuration entry
+    cap1188:
+      id: cap1188_component
+      address: 0x29
+      reset_pin: 14
+      touch_threshold: 0x40
+      allow_multiple_touches: 0x80
+
+    binary_sensor:
+      - platform: cap1188
+        id: touch_key0
+        channel: 0
+        name: "Touch Key 0"
+
+Configuration variables:
+------------------------
+
+The configuration is made up of two parts: The central component, and individual Binary sensors per channel.
+
+- **address** (*Optional*, integer): The I²C address of the sensor. Defaults to ``0x29``.
+- **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor.
+- **reset_pin** (**Required**, : ref:`pin`): Set the pin that is used to reset the CAP1188 board on boot.
+- **touch_threshold** (*Optional*, integer): The touch threshold for all channels. This defines the sensitivity for touch detection.
```

Comment:
It might be nice to add a few words here about what the allowed values for this option are, and what they mean (i.e. is low or high more sensitive?).